config/settings.yml

```diff
@@ -614,3 +614,9 @@ genisis:
   service_path: /COVID19Service
   user: TestUser
   pass: bogus
+
+# Settings for live Cerner facilities
+cerner:
+  facility_ids:
+    - 757
+    - 668
```

Comment:
This is just locally for now so I think it should be fine and will allow us to update `vets-website` to support this new behavior for the `isCerner` flag on each user's facilities